src/main/java/com/puppycrawl/tools/checkstyle/ConfigurationLoader.java

```diff
@@ -495,5 +496,18 @@ public void endElement(String uri,
                 }
             }
         }
+
+        /**
+         * Util method to recheck attribute in module.
+         * @param module modeul to check
+         * @param attributeName name of atteribute in module to find
+         * @return true if attribute is present in module
+         */
+        private boolean containsAttribute(Configuration module, String attributeName) {
```

Comment:
yes, but I do not want to extend API for now. So it is hosted only in one implementation that need it. Whole design is interesting, we have API interface, but code is not ready to have any other from DefaultConfiguration. This need to be reconsidered completely in checkstyle8.

So, for now, no extending for api.